app/assets/javascripts/application.js

```diff
@@ -40,8 +40,13 @@
 //= require controllers/ops/log_collection_form_controller
 //= require controllers/ops/tenant_form_controller
 //= require controllers/ops/tenant_quota_form_controller
+//= require controllers/container_dashboard/util/util-module
```

Comment:
what is util-module?
the name is a bit vague, and it's also under util folder, and under it there's also dashboard-utils...